Library/Homebrew/language/node.rb

```diff
@@ -32,13 +44,13 @@ def self.setup_npm_environment
       end
     end
 
-    def self.std_npm_install_args(libexec)
+    def self.std_npm_install_args(libexec, prepublish_requires_deps = false, pack_skip_prepublish = false)
```

Comment:
Minor nit, but let's shorten these arguments down and make them keywords:

```ruby
def self.std_npm_install_args(libexec, req_deps: false, skip_prepub: false)
```

(Or something along those lines).